test/unit/events_tests.js

```diff
@@ -0,0 +1,115 @@
+var test = require('../lib/ptaptest')
+var qs = require('querystring')
+var config = require('../../lib/config')
+var TestServer = require('../lib/test_server')
+var r = require('../../lib/prequest')
+
+var keys = require('../../lib/keys')(config)
+var base = 'http://' + config.server.host + ':' + config.server.port
+var publish = base + '/v0/publish'
+var events = base + '/v0/events'
```

Comment:
tiny nit: I'd suggest naming these like `publishUrl` or similar to make them clearly nouns, and make it more obvious what they contain.